src/components/environment-variables/use-poll-env-vars.js

```diff
@@ -0,0 +1,21 @@
+import { usePollingJson } from '../../effects';
+import envVarsNormaliser from '../../models/environment-variable/normaliser';
+
+const usePollEnvVars = (appName, envName, componentName, context) => {
```

Comment:
Should it be a bool parameter for context, and a more descriptive name, instead of an object? Ref https://github.com/equinor/radix-web-console/pull/356/files#r693699731

Reply:
Renamed to `poolingState`